test/unit/ITablelandController.ts

```diff
@@ -30,11 +30,12 @@ describe("ITablelandController", function () {
     accounts = await ethers.getSigners();
 
     // Deploy tables
-    tables = (await (
-      await ethers.getContractFactory("TablelandTables")
-    ).deploy()) as TablelandTables;
-    await tables.deployed();
-    await (await tables.initialize("https://foo.xyz/")).wait();
+    const Factory = await ethers.getContractFactory("TablelandTables");
+    tables = await (
+      (await upgrades.deployProxy(Factory, ["https://foo.xyz/"], {
+        kind: "uups",
+      })) as TablelandTables
+    ).deployed();
```

Comment:
All of our tests must now use the proxy deploy pattern, because it's no longer possible to call initialize on the implementation.